telegram-bot-api/ClientManager.cpp

```diff
@@ -202,14 +203,23 @@ bool ClientManager::check_flood_limits(PromisedQueryPtr &query, bool is_user_log
 }
 
 void ClientManager::get_stats(td::PromiseActor<td::BufferSlice> promise,
-                              td::vector<std::pair<td::string, td::string>> args) {
+                              td::vector<std::pair<td::string, td::string>> args,
+                              bool as_json) {
   if (close_flag_) {
     promise.set_value(td::BufferSlice("Closing"));
     return;
   }
-  size_t buf_size = 1 << 14;
+  size_t buf_size = 1 << 18;
   auto buf = td::StackAllocator::alloc(buf_size);
-  td::StringBuilder sb(buf.as_slice());
+  td::JsonBuilder jb(td::StringBuilder(buf.as_slice(), true), -1);
+  td::StringBuilder sb = std::move(jb.string_builder());
+  auto jb_root = jb.enter_object();
+  if (!as_json) {
+    jb_root.leave();
+    sb.clear();
+  } else {
+    jb_root("⚠️ WARNING", "The json representation is still a work in progress and will be changed later!");
```

Comment:
lol remove emoji

Reply:
lol, why?

Reply:
If your parser can't handle the unicode, you're out of luck with receiving text messages in telegram anyway...

Reply:
@andrew-ld so?